4_processing_features/4.extract_sc_features.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
-jupyter nbconvert --to python extract_single_cell_features.ipynb
-python extract_single_cell_features.py
+jupyter nbconvert --to python extract_sc_features_cp.ipynb
+python extract_sc_features_cp.py
+
+jupyter nbconvert --to python extract_sc_features_dp.ipynb
```

Comment:
Noticing the multiple explicit calls to `nbconvert`, would it be helpful to consider one call for all/any jupyter notebooks within this directory?

Reply:
I was actually thinking about that when I was doing it but I didn't decided to. I don't know why! I will implement this!